yoti-sdk-impl/src/test/java/com/yoti/api/client/spi/remote/AnchorConverterTest.java

```diff
@@ -0,0 +1,164 @@
+package com.yoti.api.client.spi.remote;
+
+import static java.util.Arrays.asList;
+
+import static org.hamcrest.collection.IsCollectionWithSize.hasSize;
+import static org.hamcrest.core.IsCollectionContaining.hasItem;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThat;
+
+import java.security.Security;
+import java.security.cert.X509Certificate;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Collection;
+import java.util.List;
+import java.util.Set;
+
+import com.yoti.api.client.Anchor;
+import com.yoti.api.client.Date;
+import com.yoti.api.client.SignedTimestamp;
+import com.yoti.api.client.Time;
+import com.yoti.api.client.spi.remote.proto.AttrProto;
+import com.yoti.api.client.spi.remote.util.AnchorType;
+
+import org.hamcrest.Description;
+import org.hamcrest.TypeSafeDiagnosingMatcher;
+import org.hamcrest.collection.IsCollectionWithSize;
+import org.hamcrest.core.IsCollectionContaining;
+import org.junit.Test;
+
+public class AnchorConverterTest {
+
+    static {
+        Security.addProvider(new org.bouncycastle.jce.provider.BouncyCastleProvider());
+    }
+
+    private static final String PASSPORT_ISSUER = "CN=passport-registration-server";
+    private static final String DRIVING_LICENCE_ISSUER = "CN=driving-licence-registration-server";
+    private static final String PASSPORT_SOURCE_TYPE = "PASSPORT";
+    private static final String DRIVING_LICENCE_SOURCE_TYPE = "DRIVING_LICENCE";
+    private static final String YOTI_ADMIN_VERIFIER_TYPE = "YOTI_ADMIN";
+    private static final String SHA256_ALGO = "SHA256withRSA";
+    private static final String CRITICAL_EXT_ID = "2.5.29.15";
+
+    AnchorConverter testObj = new AnchorConverter();
+
+    @Test
+    public void shouldConvertPassportAnchor() throws Exception {
+        AttrProto.Anchor anchorProto = AttrProto.Anchor.parseFrom(Base64.getDecoder().decode(TestAnchors.PP_ANCHOR));
+
+        Anchor result = testObj.convert(anchorProto);
+
+        assertEquals("SOURCE", result.getType());
+        assertEquals("OCR", result.getSubType());
+        assertEquals(PASSPORT_SOURCE_TYPE, result.getValue());
+        verifyCertificates(result.getOriginCertificates(), PASSPORT_ISSUER, AnchorType.SOURCE.extensionOid);
+        verifyTimestamp(result.getSignedTimestamp(), "2018-Apr-12 13:14:32.835537");
```

Comment:
avoid magic string or values, it's better to have is in a local or constant strings

Reply:
Ok.  I've moved these String Constants out into the TestAnchors file, and grouped them with the Anchor they are from.